plugins/node.d.midnightbsd/df_inode

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+#
+#
+# Plugin to monitor inode-usage.
+#
+# Parameters understood:
+#
+# 	config   (required)
+# 	autoconf (optional - used by munin-config)
+#
+# Magic markers (optional - used by munin-config and installation
+# scripts):
+#
+#%# family=auto
+#%# capabilities=autoconf
+
+EXCLUDEDFS="-t noprocfs,devfs,fdescfs,linprocfs,linsysfs,sysfs,nfs,nullfs,cd9660"
+
+print_values() {
+	mfs=0
+	/bin/df -P -i $EXCLUDEDFS | tail -n +2 | sort | grep -v "//" | while read i; do
+		case $i in
+		mfs:*) name=mfs$mfs; mfs=`expr $mfs + 1`;;
+		*) name=`echo $i | awk '{ gsub("[^a-zA-Z0-9_]", "_", $1); print $1 }'` ;;
+		esac
+		printf "$name.value "
+		echo $i | awk '{ total = $6 + $7; if (total == 0) print 0; else printf("%.2f\n", 100.0 * $6 / total); }'
+	done
+}
+
+if [ "$1" = "autoconf" ]; then
+	if [ "`print_values`" = "" ] ; then
```

Comment:
How about switching to `$( ... )` instead of the backticks? (if your shell supports it)